ibm/service/power/resource_ibm_pi_instance.go

```diff
@@ -220,9 +220,10 @@ func ResourceIBMPIInstance() *schema.Resource {
 				Description: "PIN Policy of the Instance",
 			},
 			helpers.PIInstanceImageId: {
-				Type:        schema.TypeString,
-				Required:    true,
-				Description: "PI instance image id",
+				Type:             schema.TypeString,
+				Required:         true,
+				Description:      "PI instance image id",
+				DiffSuppressFunc: flex.ApplyOnce,
```

Comment:
Why is the diff suprress func applied ?

Reply:
@hkantare when using a stock image the result image id from the pvm instance depends on the storage controller used to deploy the image. Here user provided configuration is not always same as the instance image id. Terraform would always show a diff to be applied which was also failing the test cases at some places depending on the storage controller selected automatically by the service provider.